app/src/main/java/com/example/dinhh/soundscape/presentation/screens/sounds/SoundFragment.kt

```diff
@@ -91,6 +92,16 @@ class SoundFragment : Fragment(), SoundAdapterViewHolderClicks {
         toggleViewHolderIcon(layoutPosition, false)
     }
 
+    override fun addSoundToSoundscape(layoutPosition: Int) {
+        selectSound(layoutPosition)
+    }
+
+
+    private fun selectSound(layoutPosition: Int){
+        val sound = adapter.getData()[layoutPosition][0]
+        soundViewModel.addSelectedSound(sound.downloadLink, sound.title, sound.length.toInt(), sound.category, 50)
```

Comment:
you can do like 

```
val soundScapeItem = SoundscapeItem(sound.dowloadlink, title, ....)

soundViewModel.addSelectedSound(soundScapeItem)
```

Reply:
Also you can set the default volume value in SoundscapeItem so you dont have to pass 50 everytime you create a SoundscapeItem

```
data class SoundscapeItem(
    val title: String,
    val length: Int,
    val category: String,
    var volume: Int = 50
)
```